server/graphql/v2/object/Expense.js

```diff
@@ -168,6 +168,12 @@ const Expense = new GraphQLObjectType({
           return req.loaders.Expense.activities.load(expense.id);
         },
       },
+      tags: {
+        type: new GraphQLList(GraphQLString),
+        resolve(expense) {
+          return expense.tags;
+        },
```

Comment:
**[:ice_cream: Details & personal preferences]** It's a detail, but could simplify things on frontend

```suggestion
        type: new GraphQLNonNull(new GraphQLList(GraphQLString)),
        resolve(expense) {
          return expense.tags || [];
        },
```